src/server/server.py

```diff
@@ -185,6 +185,7 @@
 UI_server_messages = {}
 Auth_succeeded_flag = False
 
+RESTART_FLAG = False

```

Comment:
Can we rename this SERVER_RESTART_FLAG?  Descriptive variable names help a lot with code clarity.

Reply:
Certainly. I'll tweak that and make another commit.